extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -33,6 +31,7 @@
 
 	&__counter {
 		color: $color-text;
+		padding-bottom: 12px;
```

Comment:
oh, you're totally right, I forgot about dropdown shift. Ok, lets leave it as is.